tools/walletextension/httpapi/routes.go

```diff
@@ -317,6 +321,57 @@ func networkHealthRequestHandler(walletExt *rpcapi.Services, userConn UserConn)
 	}
 }
 
+func networkConfigRequestHandler(walletExt *rpcapi.Services, userConn UserConn) {
+	// read the request
+	_, err := userConn.ReadRequest()
+	if err != nil {
+		walletExt.Logger().Error("error reading request", log.ErrKey, err)
+		return
+	}
+
+	// Call the RPC method to get the network configuration
+	networkConfig, err := walletExt.GetTenNetworkConfig()
+	if err != nil {
+		walletExt.Logger().Error("error fetching network config", log.ErrKey, err)
+	}
+
+	// Define a struct to represent the response
+	type NetworkConfigResponse struct {
```

Comment:
Yeah, it is because of that + we need to convert data fields to strings. I was also asking ChatGPT if there is a good way, but unfortunately nothing simpler than that :/